docs/unity/part-2b-c-sharp.md

```diff
@@ -334,12 +325,13 @@ public static void SendChatMessage(ReducerContext ctx, string text)
 ```
 
 ## Wrapping Up
-
+### Publishing a Module to SpacetimeDB
```

Comment:
Please add an empty line before & after the heading (I don't remember if it's strictly required by Markdown, but at least it's a convention).